api/src/firstrun.py

```diff
@@ -411,17 +411,27 @@ def create_shop_accounts_cost_centers() -> None:
 def create_shop_gift_cards() -> None:
     banner(YELLOW, "Creating Fake 'Gift Cards' and 'Gift Card & Product Mappings'")
 
-    gift_card = get_or_create(
-        GiftCard, amount=299.00, validation_code=12989519, email="test@fake.com", status="activated"
-    )
-
-    # Get existing product with ID: 64 (Makerspace access starter pack)
+    # Get existing product Makerspace access starter pack
     product = get_or_create(
         Product,
         name="Makerspace access starter pack",
     )
 
-    get_or_create(ProductGiftCardMapping, gift_card_id=gift_card.id, product_id=product.id)
+    gift_card = get_or_create(
+        GiftCard,
+        amount=product.price,
+        validation_code=12989519,
+        email="test@fake.com",
+        status="activated",
```

Comment:
What does the `activated` status mean for a gift card? Has that been used?

I would think a gift card had the following states:
- `Valid` (when it has been bought and not yet used)
- `Used` (when it has been used by a customer and cannot be used again)
- `Cancelled` (if we have paid it back to the customer for whatever reason, and the customer should not be allowed to use it)

Reply:
Cancelling would be useful, we should add that